protos/datamodel/costs.proto

```diff
@@ -0,0 +1,236 @@
+syntax = "proto3";
+
+import "datamodel/common.proto";
+
+package datamodel.v2;
+option go_package = "github.com/opencost/opencost/core/pkg/model/pb;pb";
+
+// CostMetrics represents different cost accounting methods
+// Mirrors OpenCost's multiple cost calculation approaches
+message CostMetrics {
```

Comment:
Perhaps this should be `CloudCostMetrics` to distinguish it from other costs, like `ResourceCost` below? Also, we have `...Metrics` here, but not elsewhere. Let's pick one and be consistent.